ddl/ddl_api.go

```diff
@@ -792,6 +827,54 @@ func (d *ddl) CreateTable(ctx context.Context, ident ast.Ident, colDefs []*ast.C
 	return errors.Trace(err)
 }
 
+func (d *ddl) CreateView(ctx context.Context, ident ast.Ident, colNames []string, selectFields []string, selectText string) (err error) {
+	is := d.GetInformationSchema()
+	schema, ok := is.SchemaByName(ident.Schema)
+	if !ok {
+		return infoschema.ErrDatabaseNotExists.GenByArgs(ident.Schema)
+	}
+	if is.TableExists(ident.Schema, ident.Name) {
```

Comment:
can these checks be extracted as a function,
since they are all the same as CreateTable

Reply:
OK,Done